config/packages.yml

```diff
@@ -103,6 +103,10 @@ acquia/coding-standards:
   type: phpcodesniffer-standard
   url: ../coding-standards-php
 
+acquia/drupal-environment-detector:
+  type: library
+  version_dev: "dev-master"
```

Comment:
You know what? Now that I think of it, `version_dev` isn't strictly necessary here.